albogdan/skule-vote
# Minor fixes

## Overview

- Resolves #159
- Fixes the filter for `Candidates` - removes the ability to filter by `Election`, so as to remove extra clutter
- Updated CSV templates. The new templates are more robust and contain more `Election Sessions`, `Elections`, and `Candidates` and respective `Eligilibities`

## Unit Tests Created

- N/A


## Steps to QA

- Go to `Candidates` tab in the admin site and see that the `Election` filter is not there anymore
- Go and download the new templates. See that they are correct in the CSV. 
- Create a new `Election Session` using the templates and make sure that it works.
